### DISCUSSION TITLE
Best way to add additional info to SummaryAdmin view for specific content part

### DISCUSSION THREAD

**Discussion by devlife** [OrchardCMS/OrchardCore#11654] [Q&A]
I have several content parts with specific fields that I would like to be displayed on the SummaryAdmin list. Is the best way simply to override `Content-MyPart.SummaryAdmin.cshtml`? That's easy enough but I feel like it would require additional maintenance as `Content.SummaryAdmin.cshtml` changes. 

Is there a way that I can specify a specific field should be included in the header?

**Comment by ns8482e** [comment]
There are different local zones available for summary admin shape
Meta, Tags, Content, Actions

Yoy can place Part shape from driver to target this zone for summary admin or you can use placement

**Reply by devlife** [reply]
Awesome I will check this out!

**Reply by SzymonSel** [reply]
Could you perhaps show an example how to modify the Content and Actions?
